tests/test_keyboard_shortcuts.py

```diff
@@ -0,0 +1,279 @@
+    page.show_only_unclassified_failures()
+    all_unclassified_failures = page.all_jobs
+
+    # Check number of unclassified failures
+    num_of_unclass_failures = len(all_unclassified_failures) - 1
+    rnd_number = random.randint(0, num_of_unclass_failures)
+    previous_failure = rnd_number - 1
+
+    if rnd_number == 0:
+        previous_failure = num_of_unclass_failures
+
+    # Select random unclassified failure
+    all_unclassified_failures[rnd_number].click()
+    all_unclassified_failures[previous_failure].click()
+    assumed_job_keyword = page.job_details.job_keyword_name
+
+    all_unclassified_failures[rnd_number].click()
+    page.job_details.wait_for_region_to_load()
+
+    page.select_previous_unclassified_failure()
+    page.job_details.wait_for_region_to_load()
+
+    assert page.job_details.job_keyword_name == assumed_job_keyword
+
+
+def test_select_next_info_tab_shortcut(base_url, selenium):
+    """Shortcut: 't'
+    Open Treeherder page, select random job, get active tab name, depending on
+    tab header name verify if next tab have assumed value, change tabs using
+    't' keyboard shortcut"""
+    page = TreeherderPage(selenium, base_url).open()
+    page.select_random_job()
+
+    active_tab_name = page.job_details.active_tab_name
+
+    if active_tab_name == 'Job details':
+        next_tab_name = 'Failure summary'
+        page.job_details.select_next_panel_tab()
+        assert page.job_details.active_tab_name == next_tab_name
+
+    if active_tab_name == 'Failure summary':
+        next_tab_name = 'Annotations'
+        page.job_details.select_next_panel_tab()
+        assert page.job_details.active_tab_name == next_tab_name
+
+    if active_tab_name == 'Performance':
+        next_tab_name = 'Job details'
+        page.job_details.select_next_panel_tab()
+        assert page.job_details.active_tab_name == next_tab_name
+
+
+def test_toggle_pinning_job_during_clicking_shortcut(base_url, selenium, driver):
+    """Shortcut: CTRL/CMD + job
+    Open Treeherder page, pin two random jobs, verify if number of pinned jobs
+    is greater that 1"""
+    page = TreeherderPage(selenium, base_url).open()
+    assert 0 == len(page.pinboard.jobs)
+
+    page.pin_random_job(driver)
+    page.pin_random_job(driver)
+
+    assert len(page.pinboard.jobs) > 1
+
+
+def test_pin_job_and_enter_bug_number(base_url, selenium):
+    """Shortcut: b
+    Open Treeherder page, select random job, pin job using 'b' shortcut and
+    enter bug number, do it once again, verify if there is one pinned job and
+    entered bugs are in related bugs field"""
+    page = TreeherderPage(selenium, base_url).open()
+    page.select_random_job()
+
+    bug_number = u'1729'
+    page.pin_job_and_enter_bug_number(bug_number)
+
+    bug_number = u'87539319'
+    page.pin_job_and_enter_bug_number(bug_number)
+
+    assert len(page.pinboard.jobs) == 1
+    assert bug_number in page.pinboard.related_bugs
```

Comment:
As I said above, although the shortcuts work the action of pinning a job with a bug number cannot be completed without signing in- so I would vote to remove this test.